Dockerfile

```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:1.2
 
-FROM alpine:3.15
+FROM alpine:3.14
```

Comment:
Any reason to avoid 3.15?

Reply:
Yes, the main reason of updating till 3.15 was related to original repo - they also use 3.15 BUT only for 25.X - by this reason I want to revert changes because we have issue here with OTP 25.

Reply:
That is IMHO no a valid reason. You didn't explain what problems there are with Alpine 3.15.